google/cloud/documentai_toolbox/wrappers/page.py

```diff
@@ -16,21 +16,14 @@
 """Wrappers for Document AI Page type."""
 
 import dataclasses
-from typing import cast, List, Optional, Union
+from typing import cast, List, Optional
 
 import pandas as pd
 
 from google.cloud import documentai
 from google.cloud.documentai_toolbox.constants import ElementWithLayout
 from google.cloud.documentai_toolbox.utilities import docai_utilities
 
-ChildrenElements = Union[
```

Comment:
Just a comment, no change required:

Below some of the occurrences of `ChildrenElements` are replaced with `List[ElementWithLayout]`.  This should be fine, but I want to call out the differences here:

A list of mixed type elements such as `[a_paragraph, a_block]` would be accepted as a `List[ElementWithLayout]`, but not accepted as a `ChildrenElements`.  So these two are not complete interchangeable.  (Such a list should never occur naturally.)

Reply:
That's entirely true, I just didn't think there was a clean way to notate this. Considering we need to use some kind of super class `ElementWithLayout` as the base `documentai_object`